source/base/Profiler.ooc

```diff
@@ -65,6 +70,8 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
```

Comment:
> https://github.com/cogneco/ooc-kean/issues/727

Yeah it comes right after the "optional" `This` and `this`.

There is an issue created for the method name https://github.com/cogneco/ooc-kean/issues/727 . Maybe you could leave a comment there.